docs/source/1_getting_started/hello_world.rst

```diff
@@ -293,9 +288,9 @@ The last step is to write new value "Hello, World!" into the string:
     ptr->setValueSync(dataValue);
 
 OPC UA variables contain not only values, but some additional information. The *status code* provides information about the quality of the data. If it is not **Success** the client
-can't trust the value of the variable. 
+can't trust the value of the variable.
```

Comment:
Additional there is a Source-Timestamp and a Server-Timestamp.

Reply:
Yes, it is so. However, I'd like to write about it in **Data  Access** section.